readme.md

```diff
@@ -1,4 +1,4 @@
-# yagl4j
+# frosty
```

Comment:
I just gave you (temporary) admin rights for this repo. Can you make sure the repo description and change the name.